docs/iris/src/whatsnew/3.0.3.rst

```diff
@@ -0,0 +1,587 @@
+.. include:: ../common_links.inc
+
+v3.0.3 (XX July 2021)
```

Comment:
We can update the date just before we release.

We need to do this manually in the `v3.0.x` release feature branch. Thanksfully this is now automated in `main`